server/model/block.go

```diff
@@ -161,6 +168,60 @@ func (b *Block) IsValid() error {
 	return nil
 }
 
+var safeInputPattern = regexp.MustCompile(`^[a-zA-Z0-9 _-]+$`)
+
+func ValidateFileId(id string) error {
```

Comment:
This has been implemented as well as part of PR 55, let's replace this implementation with a call to `IsValidId`

Reply:
The file ID is a bit different that then board ID. a file ID has the file extension included, that's why we have lines 182-184 here.